apim-operator/deploy/scenarios/scenario1/apim_analytics_configs.yaml

```diff
@@ -0,0 +1,39 @@
+#Copyright (c)  WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# WSO2 Inc. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: analytics-config
+  namespace: wso2-system
+data:
+  analyticsEnabled: "true"
+  analyticsSecret: "analytics-secret"
+  uploadingTimeSpanInMillis: "60000"
+  rotatingPeriod: "60000"
+  uploadFiles: "true"
+  verifyHostname: "true"
+  hostname: wso2analytics260
+  port: "9444"
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: apim-config
+  namespace: wso2-system
+data:
+  verifyHostname: "true"
```

Comment:
These artifacts are part of the deployment and used by devOps people to setup the environment.

Reply:
Removed the artifact now. (This was added to demonstrate the analytics enabled scenario with a shorter usage uploading cycle)